pie/storage/database.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+from typing import Dict, Optional, Union
+
+from sqlalchemy import BigInteger, Column, String
+
+from pie.database import database, session
+
+
+class Storage(database.base):
+    __tablename__ = "storage_data"
+
+    module = Column(String, primary_key=True)
+    guild_id = Column(BigInteger, primary_key=True)
+    key = Column(String, primary_key=True)
```

Comment:
They are all primary keys? What happens if you set the same key on multiple servers?

Reply:
It's composed primary key, so module, guild_id and key are unique. If you set same key on multiple servers AND use guild_id other than 0, it's OK.